ReactiveCocoa/Swift/Signal.swift

```diff
@@ -1025,11 +1014,9 @@ extension SignalType {
 			let disposable = CompositeDisposable()
 
 			let signalDisposable = self.observe { event in
-				switch event {
-				case .Completed:
-					break
-
-				case .Next, .Error, .Interrupted:
```

Comment:
Explicit here is good: if we ever add a `case` to `Event` this will fail to compile, and we'll be forced to make a decision for the new `case`.